anthem/lyrics/modules.py

```diff
@@ -18,12 +18,22 @@ def uninstall(ctx, module_list):
         raise AnthemError(u'Cannot uninstall modules. See the logs')
 
 
-def update_translations(ctx, module_list):
+def update_translations(ctx, module_list, overwrite=False):
     """ Update translations from module list"""
-    if not module_list:
+    if not isinstance(module_list, list):
         raise AnthemError(u"You have to provide a list of "
                           "module's name to update the translations")
+    ir_module = ctx.env['ir.module.module']
+    if hasattr(ir_module, 'update_translations'):
+        # Odoo version <= 10.0
+        method_name = 'update_translations'
+    else:
+        # Odoo version >= 11.0
+        method_name = '_update_translations'
 
-    for module in module_list:
```

Comment:
Your commit removed the iteration in order to call the method on the recordset directly, and the @p-tombez one re-integrates the iteration. Is there a reason for one or the other form?

Reply:
I missed PR of @p-tombez in first, 
we need iteration to be able use getattr and call method in dependency of version, if method name would be same we would call it on recordset

Reply:
So we should be able to use `getattr` on the whole recordset, no?
```python
>>> getattr(mods, method_name)()
```

Reply:
yes, you are correct, updated